src/checkout/model_payment_request.go

```diff
@@ -109,9 +111,9 @@ type PaymentRequest struct {
 	// The ecommerce or point-of-sale store that is processing the payment. Used in [partner arrangement integrations](https://docs.adyen.com/platforms/platforms-for-partners#route-payments) for Adyen for Platforms.
 	Store string `json:"store,omitempty"`
 	// When true and `shopperReference` is provided, the payment details will be stored.
-	StorePaymentMethod bool `json:"storePaymentMethod,omitempty"`
+	StorePaymentMethod bool `json:"storePaymentMethod"`
 	// The shopper's telephone number.
-	TelephoneNumber string `json:"telephoneNumber,omitempty"`
+	TelephoneNumber     string               `json:"telephoneNumber,omitempty"`
```

Comment:
Do we want to allow empty Strings to be sent in? Or should these always be omitted?

Reply:
Good question: the mustache templates use `omitempty` for all non-required attributes, not sure it is a great idea, especially for empty strings (I could understand for booleans). We won't change it as part of next (v6.0) release but we should look into this and decide what is best and what to change